tests/utils.go

```diff
@@ -1757,6 +1764,26 @@ func cleanNamespaces() {
 		for _, netDef := range nets.Items {
 			PanicOnError(virtCli.NetworkClient().K8sCniCncfIoV1().NetworkAttachmentDefinitions(namespace).Delete(context.Background(), netDef.GetName(), metav1.DeleteOptions{}))
 		}
+
+		// Remove Istio Sidecars
+		sidecarResource := schema.GroupVersionResource{Group: "security.istio.io", Version: "v1beta1", Resource: "peerauthentications"}
+		sidecars, err := virtCli.DynamicClient().Resource(sidecarResource).Namespace(namespace).List(context.Background(), metav1.ListOptions{})
+		if err != nil && !errors.IsNotFound(err) {
+			PanicOnError(err)
+		}
+		for _, sidecar := range sidecars.Items {
+			PanicOnError(virtCli.DynamicClient().Resource(sidecarResource).Namespace(namespace).Delete(context.Background(), sidecar.GetName(), metav1.DeleteOptions{}))
+		}
```

Comment:
You could wrap this in an helper function and simply execute it here.

Reply:
Done